app/models/doubly_linked_list.rb

```diff
@@ -29,12 +31,16 @@ def each(&block)
     head&.each(&block)
   end
 
-  def push(value)
-    if tail.nil?
-      self.head = self.tail = Node.new(value)
-    else
-      self.tail = self.tail.right = Node.new(value, tail)
+  def push(*values)
```

Comment:
Yeah, push is defined in `Array`, but I believe that it can take zero arguments or multiple arguments. (http://ruby-doc.org/core-2.3.0/Array.html#method-i-push).

I also agree that it should return the array and not the pushed value (which is why I added the `self` at the end here). I think `<<` is implemented differently than `push` and I haven't done anything with that method.